core/contracts/Voting.sol

```diff
@@ -376,31 +381,32 @@ contract Voting is Testable, MultiRole, OracleInterface {
         FixedPoint.Unsigned memory totalRewardToIssue = FixedPoint.Unsigned(0);
 
         // Loop over all price requests in the round to check for rewards.
-        PriceRequest[] storage priceRequests = round.priceRequests;
-        uint numPriceRequests = priceRequests.length;
-        for (uint i = 0; i < numPriceRequests; i++) {
+        bytes32[] storage priceRequestIds = round.priceRequestIds;
+        for (uint i = 0; i < priceRequestIds.length; i++) {
+            // Skip if the slot was deleted due to the vote being rolled over.
+            if (priceRequestIds[i] == 0) {
```

Comment:
If a vote fails to reach either threshold, the value here should've been deleted, and so we need a check here. That sounds right, right?

Reply:
Makes sense. I added a comment about the implicit check.